teacher.rb

```diff
@@ -0,0 +1,16 @@
+require './person'
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unkown')
```

Comment:
* * AS stated in the [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=Constructor%20extends%20parent%27s%20constructor%20by%20adding%20%40classroom%20and%20a%20parameter%20for%20it.) `Constructor extends parent's constructor by adding @specialization and a parameter for it.
.` for example
```ruby
def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```